Makefile

```diff
@@ -23,7 +23,7 @@ install:
 
 .PHONY: changelog
 changelog:
-	@git-chglog --next-tag $(tag) $(tag)
+	git-chglog --next-tag $(VERSION) -o CHANGELOG.md
```

Comment:
To dogfood ourselves, I think it might be better to put `build` as dependency of `changelog` target and calling `./git-chglog` instead of whatever installed somewhere in `$PATH`.

Reply:
Good suggestion. Updates pushed.

I always find the phrase "eat your own dog food" funny. I am too literal ... "But, I don't eat dog food???" 🍔